### DISCUSSION TITLE
Plugin ignores preferences and acts strange

### DISCUSSION THREAD

**Discussion by Hirnsau** [aclap-dev/video-downloadhelper#226] [Help and Q&A]
### Your question

Hi,
since the February Update videodownloadhelper doesn't work like before.
I always could see the video in the current Tab with the filesize. Now, it sometimes only shows HLS-Stream when before it showed the size just fine.

Also before, I could limit the number of downloads and it worked just fine. It downloaded one video at a time (I have limited bandwith unfortunately).
Since the update, it ignores everything and also download 10 videos at once and naturally the downloads fail.

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: de
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\F",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by paulrouget** [comment]
We're getting some reports about that queuing issue. I'm trying to reproduce and fail.

I'll investigate.

> I always could see the video in the current Tab with the filesize. Now, it sometimes only shows HLS-Stream when before it showed the size just fine.

Do you have a link to a URL that has this problem?

Also, if you go to Settings > More > Video Quality and uncheck "Ignore low quality videos", does it solve your problem?

**Reply by GrampaWildWilly** [reply]
I would also suggest an update to the latest beta version of VDH as well as an update to the latest version of the CoApp.  To find them, click the Wiki link at the top of this page.  On the Wiki page, there is a sort of Table of Contents down the right side.  Look for the relevant links & get the latest versions.
